### ISSUE TITLE
Pytorch-metric-learning losses with sentence-transformers

### ISSUE THREAD

**Issue by AlexRossi6** [UKPLab/sentence-transformers#1112]
Is it possible to use losses from https://github.com/KevinMusgrave/pytorch-metric-learning to train sentence-transformers? I would like to use multisimilarity loss

**Comment by nreimers** [comment]
Out-of-the-box it is not supported. But most common metric loss functions are already implemented in sentence-transformers.

Otherwise you can create your own loss layer. It is in most cases quite easy to create it.
